webdriver/tests/bidi/emulation/set_locale_override/conftest.py

```diff
@@ -55,3 +55,23 @@ def another_locale(default_locale, some_locale):
     raise Exception(
         f"Unexpectedly could not find locale different from the default {default_locale} and {some_locale}"
     )
+
+
+@pytest_asyncio.fixture
+async def default_accept_language(get_fetch_headers, top_context):
+    """
+    Returns default value of `Accept-Language` header.
+    """
+    headers = await get_fetch_headers(top_context)
+    return headers["Accept-Language"]
```

Comment:
As far as I got the `Accept-Language` header is optional. Should we return `None` in case it's not set?